companies-manifest.js

```diff
@@ -0,0 +1,317 @@
+    logo: MagicLeapLogo,
+  },
+  {
+    bottom: '-0.15rem',
+    height: '1.3rem',
+    key: 'https://www.moleskine.com',
+    similarWebGlobalRank: 70404,
+    logo: MoleskineLogo,
+  },
+  {
+    key: 'https://www.patreon.com/',
+    similarWebGlobalRank: 533,
+    logo: PatreonLogo,
+  },
+  {
+    bottom: '0.15rem',
+    key: 'https://www.reddit.com/',
+    similarWebGlobalRank: 20,
+    logo: RedditLogo,
+  },
+  {
+    key: 'https://seatgeek.com',
+    similarWebGlobalRank: 7292,
+    logo: SeatGeekLogo,
+  },
+  {
+    key: 'https://www.sesamegifts.com/',
+    similarWebGlobalRank: 500301,
+    logo: SesameLogo,
+  },
+  {
+    bottom: '0.4rem',
+    height: '3rem',
+    key: 'https://www.shpock.com',
+    similarWebGlobalRank: 11112,
+    logo: ShpockLogo,
+  },
+  {
+    key: 'https://www.target.com',
+    similarWebGlobalRank: 300,
+    logo: TargetLogo,
+  },
+  {
+    height: '1.7rem',
+    key: 'https://www.ticketmaster.com/',
+    similarWebGlobalRank: 1079,
+    logo: TicketmasterLogo,
+  },
+  {
+    bottom: '0.2rem',
+    key: 'https://tinder.com/',
+    similarWebGlobalRank: 561,
+    logo: TinderLogo,
+  },
+  {
+    bottom: '0.15rem',
+    height: '2.35rem',
+    key: 'https://www.typeform.com',
+    similarWebGlobalRank: 3279,
+    logo: TypeformLogo,
+  },
+  {
+    height: '2.3rem',
+    key: 'https://www.underarmour.com',
+    similarWebGlobalRank: 6221,
+    logo: UnderArmourLogo,
+  },
+  {
+    bottom: '0.3rem',
+    height: '2.1rem',
+    key: 'https://www.vimeo.com',
+    similarWebGlobalRank: 239,
+    logo: VimeoLogo,
+  },
+  {
+    height: '1.8rem',
+    key: 'https://www.vogue.com',
+    similarWebGlobalRank: 3075,
+    logo: VogueLogo,
+  },
+  {
+    bottom: '0.7rem',
+    height: '2.5rem',
+    key: 'https://www.xing.com',
+    similarWebGlobalRank: 2203,
+    logo: XingLogo,
+  },
+  {
+    height: '2.0rem',
+    key: 'https://jane.com',
+    similarWebGlobalRank: 21426,
+    logo: JaneLogo,
+  },
+  {
+    height: '2.3rem',
+    key: 'https://google.com',
+    similarWebGlobalRank: 1,
+    logo: GoogleLogo,
+  },
+  {
+    height: '1.5rem',
+    key: 'https://genius.com',
+    similarWebGlobalRank: 366,
+    logo: GeniusLogo,
+  },
+  {
+    height: '2.3rem',
+    key: 'https://www.eurostar.com',
+    similarWebGlobalRank: 11380,
+    logo: EurostarLogo,
+  },
+  {
+    bottom: '-0.2rem',
+    key: 'http://change.org/',
+    similarWebGlobalRank: 551,
+    logo: ChangeOrgLogo,
+  },
+  {
+    bottom: '0.1rem',
+    key: 'https://www.trulia.com/',
+    similarWebGlobalRank: 257,
+    logo: TruliaLogo,
+  },
+  {
+    height: '1.3rem',
+    bottom: '-0.2rem',
+    key: 'https://birchbox.com/',
+    similarWebGlobalRank: 38199,
+    logo: BirchboxLogo,
+  },
+];
+
+export default companies;
+
+module.exports = {
+  // sorting logic: the more popular a website, the higher it gets listed
+  sortedCompanies: companies.sort((a, b) => a.similarWebGlobalRank - b.similarWebGlobalRank),
```

Comment:
Let's change this to

```suggestion
export const sortedCompanies = companies.sort((a, b) => a.similarWebGlobalRank - b.similarWebGlobalRank);
```

and delete the `module.exports = {}` block